ocs_ci/ocs/ocp.py

```diff
@@ -111,7 +112,9 @@ def exec_oc_cmd(
         """
         oc_cmd = "oc "
         kubeconfig = os.getenv('KUBECONFIG')
-        if self.namespace:
+        if all_namespaces and not self.namespace:
+            oc_cmd += "-A"
```

Comment:
missing a space, should be "-A "

Reply:
line 113:
"oc " includes the space

Reply:
I think that @ebenahar mentioned space at the end not at the start.
As I see that we are adding the command right after and then we will miss the space between. `-A and command`